### DISCUSSION TITLE
Is there any version planned for Bedrock edition?

### DISCUSSION THREAD

**Discussion by stuffandnonsense** [AlexProgrammerDE/balena-minecraft-server#95] [Ideas]
I'd like to set up a server for my kids to play with their cousins in another country, but all use Bedrock edition (Android).  Do you know if there is any similar "plug and play" project for this edition, or are there any plans to make a second version or fork? The step-by-step guide here looks amazing and would really help me to get going, but I have not been able to find anything quite as clear which is compatible with the Android version.

**Comment by AlexProgrammerDE** [comment]
Hi @stuffandnonsense 
Sadly it is quits hard to get plug and play going with the original bedrock Edition. The cpu of the rpi doesn't meeting the bedrock Edition requirement. :( Buuut I got a tip for you. https://geysermc.org/ This is a plugin which creates a bridge between a java Edition server and a bedrock Edition client. All you have to do is putting it into your plugins folder and forwarding the set port in the config.yml of the plugin. But keep in mind bedrock works with udp and not tcp so you need to specificly use udp instead of tcp when forwarding the port.

**Reply by stuffandnonsense** [reply]
Thanks!  I will give it a go.  Kids would be delighted - probably happier if I just subscribed them to a commercial server, but I'd much rather start learning something at the same time!
